assets/js/googlesitekit-datastore-forms.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * Entrypoint for the core/forms data store.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ *
+ * Registers the core/forms datastore.
```

Comment:
Nitpick, but the:

```js
/**
 * Internal dependencies
 */
```

dependencies comment blocks aren't supposed to have anything else in them. Best to put this on a single-line comment above the import, like:

```js
// This import has side effects; it registers the core/forms datastore
// on the default data store (`googlesitekit.data`).

```